data/entities.json5

```diff
@@ -11356,5 +11356,14 @@
     "examples": [
         "app.pendo.io",
     ]
+  },
+  {
+    "name": "Braze",
+    "homepage": "https://www.braze.com",
+    "categories": ["analytics"],
```

Comment:
sheeh some of these are really hard to classify, huh! I guess this is analytics but also a bit of customer-success 😬 , this WFM for now